static/app/views/eventsV2/tags.spec.jsx

```diff
@@ -58,13 +63,12 @@ describe('Tags', function () {
     );
 
     // component is in loading state
-    expect(wrapper.find('StyledPlaceholder').length).toBeTruthy();
-
-    await tick();
-    wrapper.update();
+    expect(screen.getAllByTestId('loading-placeholder')[0]).toBeInTheDocument();
```

Comment:
Just out of curiosity, any reason why we should use `getAllByTestId` here instead of `getByTestId`?